### ISSUE TITLE
Cannot access Java 8 interface default method

### ISSUE THREAD

**Issue by jkkool18** [samskivert/jmustache#66]
I am unable to retrieve a getter method that is implemented as an interface default method. I believe this is because of the way DefaultCollector.getMethod() iterates over superclasses, leaving out interfaces.

Sample code:

```java
public class TestDefaultMethod {
    interface Foo {
        default String getFoo() {
            return "Foo";
        }
    }

    static class FooImpl implements Foo {};

    public static void main(String[] args) {
        FooImpl foo = new FooImpl();
        System.out.println(foo.getFoo()); // works fine
        Template tmpl = Mustache.compiler().compile("{{foo}}");
        System.out.println(tmpl.execute(foo)); // Exception! No method or field with name 'foo' on line 1.
    }
}
```

**Comment by samskivert** [comment]
Fixed in 461b108663cc10853b030137f907085c9d168543.

**Comment by jkkool18** [comment]
Awesome, you rock, thanks!
